terraform/groups/configurable-api-caller/profiles/live-eu-west-2/efs_handle_delayed_submission_sameday.json

```diff
@@ -0,0 +1,13 @@
+{
+    "API_HOST": "api.companieshouse.gov.uk",
+    "API_KEY_REF": "/api-caller/secure-key-test",
```

Comment:
check staging and live key_ref name. It's the rather unhelpfully (by me) named secure-application-key on staging/live I believe.

Reply:
fixed to right key - copy paste error. Ta